src/scripts/commands/database.ts

```diff
@@ -0,0 +1,205 @@
+// Commands related to dhis2 database direct manipulation.
+// The generated sql files can be run for example with: d2-docker run-sql <file>
+
+import _ from "lodash";
+import fs from "fs";
+import { command, subcommands, positional, flag, option, optional, string } from "cmd-ts";
+import { HierarchyLevel, OrgUnitPath } from "scripts/common";
+
+export function getCommand() {
+    const removeOrgUnitLevelCommand = command({
+        name: "remove-orgunit-level",
+        description: "Create a sql file that removes organisation units below a hierarchy level",
+        args: {
+            level: positional({
+                type: HierarchyLevel,
+                description: "Maximum hierarchy level to keep",
+            }),
+            outputFile: option({
+                type: optional(string),
+                long: "output-file",
+                description: "Name of the output file that will contain the sql commands",
+            }),
+            overwrite: flag({
+                long: "overwrite",
+                description: "Overwrite ouput file if exists",
+            }),
+        },
+        handler: async args => {
+            const outputFile = args.outputFile ?? "remove_orgunits.sql";
```

Comment:
[no strong opinion] Instead of hardcoded file paths, we can print to stdout.

Reply:
Yes, I like it better too. Done in 52f4ab802ac41ecf2049ae992b70785aa1062106 (with other things).